Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewController.swift

```diff
@@ -938,6 +942,16 @@ extension ProjectPageViewController: UITableViewDelegate {
     self.tableView.layoutIfNeeded()
   }
 
+  public func tableView(_: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+    let cell = self.dataSource.items(in: indexPath.section)[indexPath.row]
+
+    if cell.reusableId == SimilarProjectsTableViewCell.defaultReusableId {
+      return SimilarProjectsCellConstants.collectionViewHeight
+    }
+
+    return UITableView.automaticDimension
+  }
+
```

Comment:
The tableview is having trouble understanding it's content view's height when it's child is a collectionview. The best workaround I've come up with is hardcoding the collectionview height.

If any one else wants to dive deeper here with me to find a better solution I'm for it!